src/Sylius/Bundle/AdminBundle/Resources/config/routing/ajax.yml

```diff
@@ -1,15 +1,3 @@
-sylius_admin_ajax_taxon:
-    resource: "@SyliusAdminBundle/Resources/config/routing/ajax/taxon.yml"
-    prefix: /taxons
-
-sylius_admin_ajax_product:
-    resource: "@SyliusAdminBundle/Resources/config/routing/ajax/product.yml"
-    prefix: /products
-
-sylius_admin_ajax_product_attribute:
-    resource: "@SyliusAdminBundle/Resources/config/routing/ajax/product_attribute.yml"
-    prefix: /product-attributes
-
 sylius_admin_ajax_product_taxon:
     resource: "@SyliusAdminBundle/Resources/config/routing/ajax/product_taxon.yml"
     prefix: /product-taxons
```

Comment:
Can we remove these as well, or haven’t we implemented the SymfonyUX counterpart?

Reply:
this route is used in `stimulus_controller`

[src/Sylius/Bundle/AdminBundle/templates/product_taxon/grid/action/update_positions.html.twig](https://github.com/Sylius/Sylius/blob/bootstrap-shop/src/Sylius/Bundle/AdminBundle/templates/product_taxon/grid/action/update_positions.html.twig)